AUTHORS

```diff
@@ -200,25 +182,42 @@ Stefano Di Martino (StefanoD)
 Steinar Gunderson (sesse)
 Stéphane Nicolet (snicolet)
 Syine Mineta (MinetaS)
-Prokop Randáček (ProkopRandacek)
 Thanar2
-thaspel
-theo77186
 Tom Truscott
 Tom Vijlbrief (tomtor)
 Tomasz Sobczyk (Sopel97)
 Torsten Franz (torfranz, tfranzer)
 Torsten Hellwig (Torom)
 Tracey Emery (basepr1me)
-tttak
 Unai Corzo (unaiic)
 Uri Blass (uriblass)
 Vince Negri (cuddlestmonkey)
+borg323
```

Comment:
how did you order this ? Now `b` is after `v` ?

Reply:
Probably ASCII code, he put all uppercase before all lowercase.

Reply:
Ah, crap! Never trust a tool, right? Well, I guess this was ordered alphabetically, but also taking capitalization into account. Will fix this, yet again. @Disservin, thanks for spotting this. I just copy and pasted the list, trusting that the tool did the right thing.

Reply:
I have now ordered everyone in the _other Stockfish developers_ list by hand. I think it's correct now.